### PR TITLE
Add GHA job to test against all iso-10303 schemas

### PR DESCRIPTION
**Pull request by kwkwan**

### CODE REVIEW

File: lib/tasks/test_schemas.rake [hunk 1]
```diff
@@ -0,0 +1,11 @@
+require "expressir"
```
Comment (ronaldtse) [inline lib/tasks/test_schemas.rake:1]: We can just use the command line here to do it without a new Rake task:
(see https://github.com/metanorma/iso-10303/blob/main/.github/workflows/validate_express.yml)
```sh
bundle exec expressir validate schemas-srl.yml
bundle exec expressir validate schemas_module.yml
bundle exec expressir validate schemas_bom.yml
```

**Review by ronaldtse — CHANGES_REQUESTED** [review]

File: lib/tasks/test_schemas.rake (hunk 1 above)
Comment (ronaldtse) [reply lib/tasks/test_schemas.rake:1]: This is also because not all `*.exp` files in the iso-10303 repository are valid (some are not meant to be tested).

**Comment by ronaldtse** [comment]
Thank you @kwkwan !

For the schemas that are failing, can you please help create individual issues for them in Expressir (same type of failure goes in one issue)?

**Comment by kwkwan** [comment]
@ronaldtse Ok. I will create individual issues for them.

**Pull request merged**